pkg/framework/api/nodeinfo_podinfo_test.go

```diff
@@ -38,126 +36,6 @@ func makePriority(priority int32) *int32 {
 	return &priority
 }
 
-func TestPodInfoMaintainer_GetPods(t *testing.T) {
```

Comment:
Why delete this?

Reply:
I have cancelled this commit.

The reason I deleted this is because I changed GetPods to unordered for higher performance, and this test requires GetPods to be ordered.